test/test_models.py

```diff
@@ -343,12 +345,27 @@ def _check_input_backprop(model, inputs):
     _model_params[m] = {"input_shape": (1, 3, 64, 64)}
 
 
-# skip big models to reduce memory usage on CI test
+# skip big models to reduce memory usage on CI test. We can exclude combinations of (platform-system, device).
+_all_platforms = ("Darwin", "Linux", "Windows")
+_all_devices = ("cpu", "cuda")
 skipped_big_models = {
-    "vit_h_14",
-    "regnet_y_128gf",
+    "vit_h_14": set(product(_all_platforms, _all_devices)),
+    "regnet_y_128gf": set(product(_all_platforms, _all_devices)),
```

Comment:
I see we already did this before. Is this a temporary thing? If not, how are we going to make sure that we don't introduce bugs in the future?

Reply:
I have chatted with @datumbox, for vit_h_14 and regnet_y_128gf, previously it breaks on windows for both [CPU](https://github.com/pytorch/vision/issues/6189) and [GPU](https://app.circleci.com/pipelines/github/pytorch/vision/17065/workflows/85192520-7d29-4f6f-a850-8082ea4eb3f8/jobs/1382986).

I think this PR is a good idea to loosen the test for the big model so we can still test them on Linux and Mac if possible :)

I have looked at the code and it looks good to me too.

Reply:
Perfect thank for the references. I just pushed skipping only Windows for these tests as well. Let's see how this will go. If green, I'll clean up the PR and merge using the more specific skips as you both proposed.